receiver/mysqlreceiver/metadata.yaml

```diff
@@ -359,3 +406,40 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [mysqlx_threads]
+  mysql.table_open_cache:
+    enabled: true
+    description: The number of statuses (e.g. hit) for open tables cache lookups.
```

Comment:
Wording is confusing to me. Can you rephrase?

Reply:
Is it better now?